authui/src/authflowv2/components/settings-identity-email.css

```diff
@@ -0,0 +1,96 @@
+@layer components {
+  :root {
+    --settings-identity-email-large-font-size: var(
+      --typography-label-large__font-size
+    );
+    --settings-identity-email-large-font-weight: var(
+      --typography-label-large__font-weight
+    );
+    --settings-identity-email-large-line-height: var(
+      --typography-label-large__line-height
+    );
+
+    --settings-identity-email-medium-font-size: var(
+      --typography-body-medium__font-size
+    );
+    --settings-identity-email-medium-font-weight: var(
+      --typography-body-medium__font-weight
+    );
+    --settings-identity-email-medium-line-height: var(
+      --typography-body-medium__line-height
+    );
+
+    --settings-identity-inter-font-size: var(
+      --typography-title-large__font-size
+    );
+    --settings-identity-inter-font-weight: var(
+      --typography-title-large__font-weight
+    );
+    --settings-identity-inter-line-height: var(
+      --typography-title-large__line-height
+    );
+
+    --settings-identity-email-link-color: var(--body-text__link-color);
+    --setings-identity-email-success-color: var(--color-success);
+    --setings-identity-email-failure-color: var(--color-error);
+
+    --settings-identity-email-text-color: var(--color-neutral-700);
+    --settings-identity-email-border-color: var(--color-neutral-100);
+  }
+
+  :root.dark {
+    --settings-identity-email-text-color: var(--color-neutral-100);
+    --settings-identity-email-border-color: var(--color-neutral-400);
+  }
+
+  .settings-identity-email-block {
+    @apply px-2 py-4 flex flex-col;
+    border-bottom: 1px solid var(--settings-identity-email-border-color);
+  }
+
+  .settings-identity-email-indent {
+    @apply grid grid-cols-[24px_1fr_auto] gap-2;
```

Comment:
We use `rem` instead of `px`

https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units